testsuite/WIKI.md

```diff
@@ -20,20 +20,26 @@ All tests should pass (i.e. no **FAIL**, **ERROR**); *SKIPPED* or *XFAIL* are ok
 
 All tests should pass (i.e. no **FAIL**, **ERROR**); *SKIPPED* or *XFAIL* are ok. For anything that fails or gives an error **fix your code** (or *raise an issue*).
 
-**Do not push code that fails to the development branch.** Instead, push it to a feature or issue branch. It will run automatically through the unit tests by travis-ci and it will be available for comment and discussion by other developers.
-
 ## Recommended ##
 Use the tests from the [git source repository](Source), which are located in the [testsuite/MDAnalysisTests](https://github.com/MDAnalysis/mdanalysis/tree/develop/testsuite) directory:
 ```
 cd testsuite/MDAnalysisTests
+pytest  --disable-pytest-warnings
+```
+Running the tests serially can take some time (>20min) depending on the performance of your computer.
+You can also run the tests in parallel. To do so you will need `pytest-xdist` installed
+```
+pip install pytest-xdist
 pytest  --disable-pytest-warnings --numprocesses 4
 ```
+
 (Try increasing the number of processes; with 24 processes on 12 cores (+hyperthreading) this took ~40 seconds; in serial it takes ~30 min).
 
 To run specific tests just specify the path to the test file:
 ```
-pytest test_analysis.py
+pytest path_to/MDAnalysisTests/analysis/test_align.py::TestAlign::test_rmsd
 ```
+Note: You have to replace `path_to` with the actual path to where the code is.
```

Comment:
use `git_repo/testsuite/...` and give the path relative to the git repository that the user cloned.